extensions/mssql/package.json

```diff
@@ -546,12 +546,12 @@
         },
         {
           "command": "mssql.detachDatabase",
-          "when": "connectionProvider == MSSQL && nodeType == Database && !isCloud && config.workbench.enablePreviewFeatures",
+          "when": "connectionProvider == MSSQL && nodeType == Database && !isCloud && !(nodeLabel =~ /^(master|model|msdb|tempdb)$/) && config.workbench.enablePreviewFeatures",
```

Comment:
What about using nodePath and ignoring everything under the System Databases folder? See https://github.com/microsoft/azuredatastudio/blob/2baad1ff60499245f4ef212676e7da97c825b2f9/extensions/mssql/package.json#L630 for an example. That'd be much more future proof in case we have other databases we put under there.

Reply:
Switched to using nodePath

Reply:
Also excluded New for System DBs to match SSMS